app/src/androidTest/java/org/mozilla/fenix/ui/MediaNotificationTest.kt

```diff
@@ -59,7 +59,7 @@ class MediaNotificationTest {
         mockWebServer.shutdown()
     }
 
-    @Ignore("Still failing, due to https://github.com/mozilla-mobile/android-components/issues/9748")
+    @Ignore("Failing with ANR: https://github.com/mozilla-mobile/fenix/issues/15754")
     @Test
```

Comment:
I don't think we want to lose the original issue here if it's still problematic

Reply:
It's linked in the test issue, no worries.